### PR TITLE
Add option for standalone data path

### PR DESCRIPTION
**Pull request by djKooks**
Hello,
Looking on standalone mode, and seems there are options to make new standalone client by defining path or not.
(path for sled DB)

This is just a draft, so please confirm following feature is reasonable or not, and I will go on to the next step.

🙏

### CODE REVIEW

**Review by edrevo — COMMENTED** [review]
Looks like a good start! I wonder if sled needs to be properly closed (I'm on a cell phone right now) in order to ensure that the file isn't corrupt. Right now we don't have any kind of graceful shutdown on the scheduler, so if sled needs to be flushed/closed, we would need this graceful shutdown for this change to be useful

**Comment by djKooks** [comment]
@edrevo I've checked sled, and seems it does not require `close` handler after usage.

**Comment by djKooks** [comment]
@edrevo how do you think of input name? Is `standalone_data_path` seems okay?

File: rust/executor/executor_config_spec.toml [hunk 1]
```diff
@@ -59,4 +59,9 @@ abbr = "c"
 name = "concurrent_tasks"
 type = "usize"
 default = "4"
-doc = "Max concurrent tasks."
\ No newline at end of file
+doc = "Max concurrent tasks."
+
+[[param]]
+name = "standalone_data_path"
```
Comment (andygrove) [inline rust/executor/executor_config_spec.toml:65]: Perhaps a better name could be `scheduler_data_path` or something similar?

Comment (djKooks) [reply rust/executor/executor_config_spec.toml:65]: @andygrove seems good. But as I know this parameter will be only used under `standalone` mode. Will it not confuse?

**Comment by djKooks** [comment]
@andygrove renamed~please check

**Comment by andygrove** [comment]
I am closing this PR and archiving this repository. Development will continue over in Apache Arrow now.

**Pull request closed without merging**